File: Data Structures & Algorithms/top-k-elements-in-list/submission-1.cpp
bool cmp(pair<int, int>& a, 
        pair<int, int>& b) 
    { 
        return a.second > b.second; 
    }
class Solution {
    
public:
     
    vector<int> topKFrequent(vector<int>& nums, int k) {
        unordered_map<int,int>seen;
        vector<int>sol;
        for(int i=0;i<nums.size();i++){
            int s= seen.count(nums[i]);
            if(s==0)seen[nums[i]]=1;
            else seen[nums[i]]+=1;
        }
        vector<pair<int,int>>sot;
        for(auto it:seen){
            sot.push_back(it);
        }
        sort(sot.begin(),sot.end(),cmp);

        int i=0;
        vector<pair<int,int>>::iterator it = sot.begin();
        while(i<k){
            sol.push_back(it->first);
            it++;
            i++;
        }

        return sol;

    }
};

File: Data Structures & Algorithms/two-integer-sum/submission-1.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        ls=[]
        for i in range(len(nums)):
            for j in range(len(nums)):
                if i==j :
                    continue
                else:
                    if nums[i]+nums[j]==target:
                        ls.append(i)
                        ls.append(j)
                        return ls
                        break